packages/scenario/test/submission.test.ts

```diff
@@ -156,6 +156,116 @@ describe('Form submission', () => {
 			);
 		});
 
+		describe('instance root attributes', () => {
+			it('serializes attributes from the instance root, as they appear in the form definition', async () => {
+				const formId = 'inst-root-attrs';
+				const version = '2018061801';
+
+				const scenario = await Scenario.init(
+					'XML serialization - instance root attrs',
+					// prettier-ignore
+					html(
+						head(
+							title('XML serialization - instance root attrs'),
+							model(
+								mainInstance(
+									t(`data id="${formId}" version="${version}"`,
+										t('inp', 'val'),
+										t('meta',
+											t('instanceID')))
+								),
+								bind('/data/inp').type('string'),
+								bind('/data/meta/instanceID').calculate(`'${DEFAULT_INSTANCE_ID}'`)
+							)
+						),
+						body(
+							input('/data/inp', label('Input (with default value)'))
+						)
+					)
+				);
+
+				expect(scenario).toHaveSerializedSubmissionXML(
+					// prettier-ignore
+					t(`data id="${formId}" version="${version}"`,
+						t('inp', 'val'),
+						t('meta',
+							t('instanceID', DEFAULT_INSTANCE_ID))).asXml()
+				);
+			});
+
+			it('preserves instance root attribute namespace prefixes, as they appear in the form definition', async () => {
```

Comment:
Is the namespace declaration also included in the serialization? That could be layered on, I think, but is important too.